swig/python/CMakeLists.txt

```diff
@@ -80,9 +80,14 @@ else ()
       ${CMAKE_CURRENT_BINARY_DIR}/osgeo/gnm.py
       ${CMAKE_CURRENT_BINARY_DIR}/osgeo/ogr.py
       ${CMAKE_CURRENT_BINARY_DIR}/osgeo/osr.py)
-  file(TO_NATIVE_PATH "${CMAKE_CURRENT_BINARY_DIR}/osgeo/" OSGEO_BINARY_DIR)
-  file(TO_NATIVE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/osgeo/" OSGEO_SOURCE_DIR)
-  if (Python_NumPy_FOUND)
+  if(WIN32)
+    file(TO_NATIVE_PATH "${CMAKE_CURRENT_BINARY_DIR}\\osgeo\\" OSGEO_BINARY_DIR)
```

Comment:
that if() else() doesn't make sense to me. TO_NATIVE_PATH should do the change from slash to backslash on Windows

Reply:
I check the original source code using MinGW environment. However, if "/" is applied, the installation is wrong. 
mkdir build
cd build 
cmake .. -DPROJ_LIBRARY="%cd%/../../lib_gdal/build/third-party/install/lib/libproj.dll.a" -DPROJ_INCLUDE_DIR="%cd%/../../lib_gdal/build/third-party/install/include" -DSQLite3_LIBRARY="%cd%/../../lib_gdal/build/third-party/install/lib/libsqlite3.dll.a" -DSQLite3_INCLUDE_DIR="%cd%/../../lib_gdal/build/third-party/install/include" -DGDAL_USE_HDF5=OFF -DGDAL_USE_BLOSC=OFF -DGDAL_USE_ODBC=OFF -DCMAKE_INSTALL_PREFIX="%cd%\..\installed" -G "CodeBlocks - MinGW Makefiles"

Consequently, the installed file is not located in "%cd%\..\installed" but in ${CMAKE_BINARY_DIR}/swig/python/%cd%/../installed. 
That is, it ignore the the backslash "E:/xxx"  (E:="%cd") and regard it as "E:xxx".  When applying the file(TO_NATIVE_PATH "${CMAKE_CURRENT_BINARY_DIR}\\osgeo\\" OSGEO_BINARY_DIR), the above error disappear and the installation is correct.